web/modules/custom/rdf_entity/rdf_entity.module

```diff
@@ -398,3 +400,86 @@ function rdf_entity_theme_suggestions_rdf_entity(array $variables) {
 
   return $suggestions;
 }
+
+/**
+ * Returns the requirements related to virtuoso version.
+ *
+ * @return array
+ *   The virtuoso version requirements.
+ */
+function rdf_entity_virtuoso_version_requirements() {
+  $minimum_version = '07.00.0000';
+  /** @var \Drupal\rdf_entity\Database\Driver\sparql\Connection $connection */
+  $connection = Database::getConnection('default', 'sparql_default');
+  $client = Http::getDefaultHttpClient();
+  $client->resetParameters(TRUE);
+  $client->setUri($connection->getQueryUri());
+  $client->setMethod('GET');
+  $response = $client->request();
```

Comment:
I have shut off my Sparql instance to test the other requirement function, but I got a critical error here because it tries to connect to Sparql to get the version:

> EasyRdf\Exception: Unable to connect to localhost:8890 (Connection refused) in /Users/francesco/Projects/web/joinup/vendor/easyrdf/easyrdf/lib/Http/Client.php:440        [error]
Stack trace:
#0 /Users/francesco/Projects/web/joinup/web/modules/custom/rdf_entity/rdf_entity.module(418): EasyRdf\Http\Client->request()